### PR TITLE
Integrate stackstate agent integrations

### PR DESCRIPTION
**Pull request by craffit**
### What does this PR do?

- incorporate new stackstate-agent-integrations repo

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: tasks/agent.py [hunk 1]
```diff
@@ -207,6 +207,19 @@ def apply_branding(ctx):
...
     do_sed_rename(ctx, camel_replace, "./cmd/agent/gui/views/private/js/javascript.js")
 
+    # stackstate_checks
+    do_go_rename(ctx, '"\\"datadog_checks\\" -> \\"stackstate_checks\\""', "./cmd/agent/app")
+    do_sed_rename(ctx, 's/datadog_checks_base/stackstate_checks_base/g', "cmd/agent/app/integrations.go")
+    do_go_rename(ctx, '"\\"datadog_checks\\" -> \\"stackstate_checks\\""', "./pkg/collector/py")
+
+    # This part we renamed in-tree because it is used by unit tests and we do not apply branding before unit tests,
+    # nor do we want to when developing. We do want to check all renames are there though, whihc is what this code does.
+    grepResult = ctx.run('grep  -R datadog_checks ./cmd/agent/dist/', warn=True)
```
Comment (TizianoPerrucci) [inline tasks/agent.py]: this is a bit unexpected to find it here, we could move this to `test/renaming` and do a package wide check

File: omnibus/config/software/stackstate-agent-integrations.rb [hunk 2]
```diff
@@ -153,12 +148,12 @@
 
     # Install the static environment requirements that the Agent and all checks will use
     if windows?
-      command("#{python_bin} -m #{python_pip_no_deps}\\datadog_checks_base")
-      #[VS] command("#{python_bin} -m #{python_pip_no_deps}\\datadog_checks_downloader --install-option=\"--install-scripts=#{windows_safe_path(install_dir)}/bin\"")
+      command("#{python_bin} -m #{python_pip_no_deps}\\stackstate_checks_base")
+      #[VS] command("#{python_bin} -m #{python_pip_no_deps}\\stackstate_checks_downloader --install-option=\"--install-scripts=#{windows_safe_path(install_dir)}/bin\"")
```
Comment (craffit) [inline omnibus/config/software/stackstate-agent-integrations.rb]: ditch this

Comment (craffit) [reply omnibus/config/software/stackstate-agent-integrations.rb]: discuss with mark downloading/security

Comment (craffit) [reply omnibus/config/software/stackstate-agent-integrations.rb]: we discussed that for now we ship checks with the agent, and deb/rpm signing gives enough security

File: tasks/agent.py (hunk 1 above)
Comment (craffit) [reply tasks/agent.py]: fixed

**Pull request merged**